cloudpickle/cloudpickle.py

```diff
@@ -238,7 +238,11 @@ def dump(self, obj):
 
     def save_memoryview(self, obj):
         """Fallback to save_string"""
-        Pickler.save_string(self, str(obj))
+        try:
+            arg = obj.obj
+        except ValueError:
+            arg = _released_memory_view_sentinel
+        self.save_reduce(_fill_memoryview, (arg,), obj=obj)
```

Comment:
good catch! It looks like there is no way in the Python API to get the offset of the view, how would people feel about an optional C extension to get the offset and stride information?